graphene/src/operation.rs

```diff
@@ -215,8 +215,8 @@ impl Operation {
 	/// # Safety
 	/// This function reads from uninitialized memory!!!
 	/// Only use if you know what you are doing
-	unsafe fn as_slice(&self) -> &[u8] {
-		core::slice::from_raw_parts(self as *const Operation as *const u8, std::mem::size_of::<Operation>())
+	unsafe fn as_slice(&self) -> &[u32] {
+		core::slice::from_raw_parts(self as *const Operation as *const u32, std::mem::size_of::<Operation>())
```

Comment:
Why are we changing this?

Reply:
Just tested it again after reverting this change and feature still works as intened, committing the revert